docs/angular/guides/setup-mfe-with-angular.md

```diff
@@ -0,0 +1,393 @@
+# How to setup a Micro Frontend with Angular and Nx
+
+[Webpack 5]() introduced a [Module Federation Plugin]() enabling multiple, independently built and deployed bundles of code to form a single application. This is the foundation of Micro Frontend Architecture and the Module Federation Plugin makes implementing such an architecture much simpler.  
+With Angular 12 adding support for Webpack 5 it increases the viability of scaffolding a Micro Frontend architecture with Angular.
+
+There are some challenges that exist, such as being able to be able to pass a custom Webpack configuration file, which is required to add the Module Federation configuration, to Angular's compiler.
+
+We recently made changes to our `@nrwl/angular:webpack-browser` executor that allows you to supply a path to a custom Webpack configuration file which we then merge with Angular's default Webpack configuration before telling Angular to use this merged configuration to build the Angular application.
```

Comment:
I don't think we need a story here.

In 3 months, it won't be so recent anymore :stuck_out_tongue:

Not sure we need this paragraph at all since it will be included in the guide later?

Reply:
Sure thing! Can keep it straight to the point.